lib/galaxy/webapps/galaxy/services/history_contents.py

```diff
@@ -155,6 +156,11 @@ class LegacyHistoryContentsIndexParams(Model):
     dataset_details: Optional[DatasetDetailsType]
     deleted: Optional[bool]
     visible: Optional[bool]
+    sharable: Optional[bool] = Field(
```

Comment:
I'm a bit confused here, I thought that the legacy version of the endpoint would be deprecated and we want to add this new parameter to the *latest* version of the index endpoint in `HistoryContentsIndexParams`.

Reply:
Do you still think this is the case? The things called legacy seem to be the default way the API is used? You need to flag something special to take the non-legacy path.

Reply:
Hmmm... :thinking: That's a good point :laughing: 
The UI is using the other `dev` versions of the endpoint, they seem to differ mainly in how content is filtered, so I guess it makes sense to add this to the default legacy path.